docs/source/futures.rst

```diff
@@ -5,10 +5,6 @@
 torch.futures
 =============
 
-.. warning::
-  The ``torch.futures`` package is experimental and subject to change.
```

Comment:
I'm proposing to remove it as I suspect this is "stale": the Future class is used by the RPC API, which is now stable, hence I suppose the Future API is stable too?